build.xml

```diff
@@ -112,6 +112,19 @@
         </and>
     </condition>
 
+    <condition property="settings.dir" value="${user.home}/.jmri">
```

Comment:
This works:
`<condition property="settings.dir" value="${user.home}/JMRI">`

Note the dot needs to be removed before JMRI.

Reply:
Done!  Thanks.